mwdb/model/file.py

```diff
@@ -47,6 +49,14 @@ def get(cls, identifier):
             File.sha512 == identifier,
             File.md5 == identifier))
 
+    @property
+    def upload_stream(self):
+        return getattr(self, "_upload_stream", None)
+
+    @upload_stream.setter
+    def upload_stream(self, stream):
+        setattr(self, "_upload_stream", stream)
```

Comment:
why not:
```suggestion
        self._upload_stream = stream
```

Reply:
Linters will say that "`_upload_stream` is not initialized in `__init__`". I don't really want to mess with `__init__` because it's ORM object.